bugnote_view_inc.php

```diff
@@ -329,21 +332,17 @@
 				echo string_display_links( $t_activity['note']->note );
 				$t_add_space = true;
 			}
-		} else {
-			if ( !$t_security_token_attachments_delete ) {
-				$t_security_token_attachments_delete = form_security_token( 'bug_file_delete' );
-			}
-
-			print_bug_attachment( $t_activity['attachment'], $t_security_token_attachments_delete );
 		}
 
 		if( isset( $t_activity['attachments'] ) && count( $t_activity['attachments'] ) > 0 ) {
-			if ( !$t_security_token_attachments_delete ) {
-				$t_security_token_attachments_delete = form_security_token( 'bug_file_delete' );
-			}
-
-			foreach( $t_activity['attachments'] as $t_attachment ) {
-				print_bug_attachment( $t_attachment, $t_security_token_attachments_delete );
+			if( $t_activity['type'] === ENTRY_TYPE_NOTE ) {
```

Comment:
Remove the if - condition is always true, since we exclude other activity types at the beginning of the main loop (line 147)